lib/background.js

```diff
@@ -5,6 +5,7 @@
 chrome.webRequest.onHeadersReceived.addListener(details => {
   if (details.statusCode === 404 &&
       details.frameId === 0 &&
+      details.type == "main_frame" &&
```

Comment:
So, I thought that would have been covered by the frameId === 0 on the line above, as per [Chrome's API docs](https://developer.chrome.com/extensions/webRequest#property-details-frameId)…
I'll merge it for now, but I think we might need to file a bug.